.eslintrc

```diff
@@ -115,7 +115,6 @@
     "no-undef-init": 2,
     "no-undef": 1,
     "no-unused-vars": 1,
-    "no-use-before-define": 1,
```

Comment:
I had a look at http://eslint.org/docs/rules/no-use-before-define
Where it's not relevant to ES6 anymore. (also mentioned this in my commit message)

This rule gets really annoying when writing types.